.github/workflows/validate_pyinstaller.yml

```diff
@@ -0,0 +1,59 @@
+name: Validate Pyinstaller Build
+
+on:
+  pull_request:
+    branches:
+      - develop
+
+jobs:
+  build-for-linux:
+    name: build-pyinstaller-linux
+    runs-on: ubuntu-latest
```

Comment:
can we change this to be many linux? or containerized? for 1:1 parity?

Reply:
We are now running in `quay.io/pypa/manylinux2014_x86_64:latest` image